CHANGELOG.md

```diff
@@ -6,6 +6,17 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ## [Unreleased]
 
+## [2.2.0] - 2018-11-27
```

Comment:
This formatting (the `[]` around the version) is used so we can include a link at the bottom for the comparison between versions, like so:

```markdown
[2.2.0]: https://github.com/puppetlabs/puppetlabs-sqlserver/compare/2.1.0..2.2.0
```

Note that in this case, the tag doesn't exist yet (`2.2.0`) but will after release.

Reply:
NB: I think we miss this a lot on our modules, it's easy to forget.